src/components/InAantallen.vue

```diff
@@ -23,9 +20,6 @@
 
 <script>
 import { mapGetters } from 'vuex'
-
-import tooltip from './Tooltip'
-
```

Comment:
See remarks about bootstrap, tooltip should be included

Reply:
tooltip is back